### PR TITLE
Fix note about SA OIDC discovery doc default binding

### PR DESCRIPTION
**Pull request by enj**
Added in v1.19: https://pr.k8s.io/88344

Signed-off-by: Monis Khan <mok@vmware.com>

/assign @mtaufen

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-service-account.md [hunk 1]
```diff
@@ -349,8 +349,11 @@ JSON Web Key Set (JWKS) at `/openid/v1/jwks`. The OpenID Provider Configuration
 is sometimes referred to as the _discovery document_.
 
 Clusters include a default RBAC ClusterRole called
-`system:service-account-issuer-discovery`. No role bindings are provided
-by default. Administrators may, for example, choose whether to bind the role to
+`system:service-account-issuer-discovery`. A default RBAC ClusterRoleBinding
+assigns this role to the `system:serviceaccounts` group, which all service
+accounts implicitly belong to. This allows pods running on the cluster to access
+the service account discovery document via their mounted service account token.
+Administrators may, for example, choose whether to bind the role to
```
Comment (mtaufen) [inline content/en/docs/tasks/configure-pod-container/configure-service-account.md:356]: suggestion:
`Administrators may, additionally, choose to bind the role to ...`

Comment (enj) [reply content/en/docs/tasks/configure-pod-container/configure-service-account.md:356]: Fixed.

**Pull request merged**